client/my-sites/themes/theme-showcase.jsx

```diff
@@ -198,6 +204,18 @@ class ThemeShowcase extends React.Component {
 				.sort( ( a, b ) => a.order - b.order )
 		);
 
+		const SampleBanner = () => (
+			<ThemesBanner
+				title={ translate( 'Are you a small business?' ) }
+				description={ translate(
+					"We understand your time contraints, and that's why we created this easy-to-set-up Premium theme called Small Business."
+				) }
+				ctaLabel={ translate( 'Learn about this theme' ) }
+				ctaAction={ this.helloWorld }
+				backgroundImage={ '/calypso/images/themes-banner/banner-bg.jpg' }
```

Comment:
For string literal props, simply use double quotes (instead of curly braces plus single quotes):

```js
backgroundImage="/calypso/images/themes-banner/banner-bg.jpg"
```